### PR TITLE
RFC: Coprocessor Cache

### PR DESCRIPTION
**Pull request by breeswish**
Signed-off-by: Breezewish <breezewish@pingcap.com>

### CODE REVIEW

**Comment by andylokandy** [comment]
@breeswish Can we merge the RFC now?

**Comment by breeswish** [comment]
Yes, as long as there are enough approvals. @andylokandy

**Review by andylokandy — APPROVED** [review]
LGTM since it's implemented and delivered 🙃

File: text/2020-04-22-coprocessor-cache.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+   The cache key is the Coprocessor request data without the Ts.
+
+   The cache is a LRU cache.
+
+3. On TiDB side, before sending Coprocessor request, CoprocessorCache is checked
+   first.
+
+   TiDB sends the Coprocessor request with `IsCacheEnabled` and
+   `RegionRaftApplyIndex` field hen the following condition is met:
```
Comment (BusyJay) [inline text/2020-04-22-coprocessor-cache.md:81]: What is "hen"?

**Review by zhangjinpeng1987 — APPROVED** [review]
LGTM

**Pull request merged**